### DISCUSSION TITLE
How to re-join the #zed channel?

### DISCUSSION THREAD

**Discussion by 13tales** [zed-industries/zed#15952] [Feature Requests]
I've recently started experimenting with Zed's collaboration features. At some point I noticed the public Zed dev channel had been automatically added to my collaboration panel; in the moment I was a little annoyed by that, so I left the channel.

I've since decided I'd actually like to re-join it to stay in touch with Zed's development and get an idea of how the Zed team use the collaboration features, but I can't see a way to re-join the channel 😅 

Any suggestions?

**Comment by sleeping-at-night** [comment]
Does the solution in https://github.com/zed-industries/zed/discussions/14708 help?

**Reply by 13tales** [reply]
Thanks, that's done it 👍 

I'd somehow missed that answer in my search 😰
